src/components/AttentionBox/AttentionBox.tsx

```diff
@@ -1,13 +1,13 @@
 import React, { useMemo } from "react";
 import cx from "classnames";
-import Icon, { IconSubComponentProps } from "../Icon/Icon";
+import Icon from "../Icon/Icon";
 import IconButton from "../IconButton/IconButton";
 import CloseSmall from "../Icon/Icons/components/CloseSmall";
 import AlertIcon from "../Icon/Icons/components/Alert";
 import { IconType } from "../Icon/IconConstants";
 import { backwardCompatibilityForProperties } from "../../helpers/backwardCompatibilityForProperties";
 import { AttentionBoxType } from "./AttentionBoxConstants";
-import VibeComponentProps from "src/types/VibeComponentProps";
+import { SubIcon, VibeComponentProps } from "src/types";
```

Comment:
This kills internal packages that consume Vibe, import relatively, same for all other instances.

Reply:
I agree, but what interesting is how it's works before with:
```
import VibeComponentProps from "src/types/VibeComponentProps";
```